.github/workflows/patch-release-build.yml

```diff
@@ -0,0 +1,99 @@
+      - id: checkout-release-branch
+        name: Check out release branch
+        continue-on-error: true
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ steps.parse-release-branch.outputs.release-branch-name }}
+      - id: checkout-release-tag
+        name: Check out release tag
+        if: ${{ steps.checkout-release-branch.outcome == 'failure' }}
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ steps.parse-release-branch.outputs.release-tag-name }}
+      - name: Create release branch
+        if: ${{ steps.checkout-release-tag.outcome == 'success' }}
+        run: |
+          git checkout -b ${{ steps.parse-release-branch.outputs.release-branch-name }}
+          git push --set-upstream origin ${{ steps.parse-release-branch.outputs.release-branch-name }}
+  build:
+    runs-on: ubuntu-latest
+    needs: prepare-release-branch
+    steps:
+      - name: Checkout release branch
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ needs.prepare-release-branch.outputs.release-branch-name }}
+          submodules: true
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Setup git name
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Cherrypicks
+        run: |
+          git fetch origin master
+          echo ${{ github.event.inputs.commits }} | sed -n 1'p' | tr ',' '\n' | while read word; do
+              # Trim whitespaces and cherrypick
+              echo $word | sed 's/ *$//g' | sed 's/^ *//g' | git cherry-pick --stdin
+          done
+      - uses: burrunan/gradle-cache-action@v1.5
+        with:
+          job-id: jdk11
+          remote-build-cache-proxy-enabled: false
+          arguments: build --stacktrace -Prelease.version=${{ github.event.inputs.version }}
+          properties: |
+            enable.docker.tests=true
+      - name: Publish artifacts
+        uses: burrunan/gradle-cache-action@v1.5
+        with:
+          job-id: jdk11
+          remote-build-cache-proxy-enabled: false
+          arguments: final --stacktrace -Prelease.version=${{ github.event.inputs.version }}
+          properties: |
+            enable.docker.tests=trued
+        env:
+          CI: true
+          BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+          BINTRAY_KEY: ${{ secrets.BINTRAY_KEY }}
+          GRGIT_USER: ${{ github.actor }}
+          GRGIT_PASS: ${{ secrets.GITHUB_TOKEN }}
+      - run: git push
+      - name: Checkout master
+        uses: actions/checkout@v2
+        with:
+           submodules: true
+      - uses: burrunan/gradle-cache-action@v1.5
+        with:
+          job-id: jdk11
+          remote-build-cache-proxy-enabled: false
+          arguments: prepareRelease --stacktrace -Prelease.version=${{ github.event.inputs.version }}
```

Comment:
What does this do after `final` task above?

Reply:
Yeah I should just rename to `updateReadme` instead of `prepareRelease`, I didn't notice this use case of needing to do it on a different branch from the release at the time. Should I do it in this PR?

Reply:
I think so